client/landing/site.landing/styl/team.creation.styl

```diff
@@ -14,9 +14,6 @@ team.group.creation.styl
 
   .kdtabpaneview .TeamsModal.TeamsModal--groupCreation
 
-    &.alreadyMember:after
```

Comment:
This line is replaced with `with-avatar` class (see changes in `team.styl`). If teams modal has `alreadyMember` class, it also has `with-avatar` class